requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask==1.1.2
+Flask-SQLAlchemy==2.4.4
+psycopg2-binary==2.8.5
```

Comment:
did you use `pip freeze` to create this file? The list here seems too short from what I know of Flask and SQLAlchemy's dependencies....

Reply:
No, I just add the dependencies. Will be fixed.